test/core/test_float_builtins.c

```diff
@@ -34,6 +34,7 @@ int test_builtins() {
   TEST(fmin)
   TEST(fmod)
   TEST(scalbn)
```

Comment:
`scalbn` was already in this test it seems, so I was wondering why didn't we see an error before, and looks like we don't run the test in LTO mode. Maybe we should add a variation on the test for that?

Reply:
We do run the entire `lto2` test suite on the emscripten-releases builder.

But I will add this specific test to the github CI too.

Reply:
lto2.test_float_builtins passes fine on main .. so I guess its not enough to just reference scaldnf.   This is why the repo cases requires ldexp.. I'm not 100% why this is.. but something to do with the fact the symbol gets indirectly pulled in...